tequilapi/api_test.go

```diff
@@ -16,7 +16,7 @@ type tequilapiTestSuite struct {
 func (testSuite *tequilapiTestSuite) SetupSuite() {
 	var err error
 	idmFake := identity.NewIdentityManagerFake()
-	testSuite.server, err = StartNewServer("localhost", 0, NewApiEndpoints(idmFake))
+	testSuite.server, err = StartNewServer("localhost", 0, NewApiEndpoints(idmFake, nil))
```

Comment:
`nil`, You cat get segmentation error like that

Reply:
Noted. It also points out that in order just to check simple endpointer we need a lot of fake depencies because all endpoints are registered. Will refactor to separate functions for endpoint registering:
router :=NewApiRouter()
registerIdentitiesEndpoint(router , <identities endpoint deps>
registerConnectionEndpoint(router, <connection endpoint deps>